README.md

```diff
@@ -193,9 +193,10 @@ Advanced usage:
     {outputName}      The output file's filename without extension
     {outputExt}       The output file's extension
 
-    {pocket}  The ROM's core-specific /Assets/* directory for the Analogue Pocket (e.g. "gb")
-    {mister}  The ROM's core-specific /games/* directory for the MiSTer FPGA (e.g. "Gameboy")
-    {onion}   The ROM's emulator-specific /Roms/* directory for OnionOS/GarlicOS (e.g. "GB")
+    {pocket}    The ROM's core-specific /Assets/* directory for the Analogue Pocket (e.g. "gb")
+    {mister}    The ROM's core-specific /games/* directory for the MiSTer FPGA (e.g. "Gameboy")
+    {onion}     The ROM's emulator-specific /Roms/* directory for OnionOS/GarlicOS (e.g. "GB")
+    {batocera}  The ROM's emulator-specific /roms/* directory for Batocera (e.g. "gb")
```

Comment:
Everything in here looks great @dcramer, I'm happy to add these and then extend more in a future PR.

If you could do me a favor and revert `README.md` and leave it with no changes, I have a convoluted `scripts/update-readme-help.sh` script that updates the README at the time of version bumping, so that the README always reflects the capabilities of the published version. I realize this isn't obvious, so I'm very open to suggestions on better guardrails.